dora/core/common/src/main/java/alluxio/grpc/GrpcServerBuilder.java

```diff
@@ -88,6 +88,19 @@ public static GrpcServerBuilder forAddress(GrpcServerAddress serverAddress,
     return new GrpcServerBuilder(serverAddress, null, conf);
   }
 
+  /**
+   * Create a new instance of {@link GrpcServerBuilder} with authentication support.
+   *
+   * @param serverAddress server address
+   * @param authenticationServer the authentication server to use
+   * @param conf the Alluxio configuration
+   * @return a new instance of {@link GrpcServerBuilder}
+   */
+  public static GrpcServerBuilder forAddress(GrpcServerAddress serverAddress,
+      AuthenticationServer authenticationServer, AlluxioConfiguration conf) {
+    return new GrpcServerBuilder(serverAddress, authenticationServer, conf);
```

Comment:
why do not change the construct method to public?

Reply:
this is following how the other functions works.